src/tir/schedule/analysis/layout.cc

```diff
@@ -167,20 +167,25 @@ Optional<IndexMap> SuggestIndexMap(const Buffer& buffer, const Array<PrimExpr>&
     }
     return a.lower_factor > b.lower_factor;
   });
+  // Compute the inverse permutation by argsort
+  std::vector<int> inverse_order = order;
```

Comment:
Shall we add a test for `Suggest-Index-Map` where the inverse of the mapping is not detectable?